dbf2csv/models/dbf2csv.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 INGETIVE (<https://ingetive.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import csv
+import base64
+from odoo import models, fields, api
+from dbfpy import dbf
+
+
+class FileDBF(models.TransientModel):
+    _name = 'dbftocsv.filedbf'
+    _description = 'File DBF to convert to CSV'
+
+    data_file = fields.Binary(
+        string='DBF file',
+        required=True,
+        help='Get you DBF file.')
+    filename = fields.Char(string='DBF file name')
+
+    @api.multi
+    def proccess_file(self):
+        # Process the DBF file chosen in the wizard,
+        # create and download CSV file.
+        self.ensure_one()
+        # Create DBF file in TMP
+        fp = open('/tmp/' + self.filename, 'w+')
+        fp.write(base64.decodestring(self.data_file))
+        fp.close()
+
+        with open('/tmp/' + self.filename.replace('dbf', 'csv'),
```

Comment:
Use [`os.path.join`](https://docs.python.org/2/library/os.path.html#os.path.join) to join path components and [`os.path.splitext`](https://docs.python.org/2/library/os.path.html#os.path.splitext) to change extension more safely ([example](https://stackoverflow.com/a/7363015/)).